src/QsFmt/Formatter/SyntaxTree/Statement.fs

```diff
@@ -50,6 +50,17 @@ module QubitBinding =
     let mapPrefix mapper (binding: QubitBinding) =
         { binding with Name = SymbolBinding.mapPrefix mapper binding.Name }
 
+type ForBinding =
+    {
+        Name: SymbolBinding
+        In: Terminal
+        Value: Expression
+    }
+
+module ForBinding =
+    let mapPrefix mapper (binding: ForBinding) =
+        { binding with Name = SymbolBinding.mapPrefix mapper binding.Name }
```

Comment:
This probably already handles `_` as the symbol binding, but I'm not sure how to check. Can you confirm if that case is handled properly?

Reply:
Sorry for the late response. I've just double-checked. The `_` symbol is treated as any other symbol binding, so there is no special logic needed for handling that case.